services/parser-database/app.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+from flask import Flask  # pylint: disable=import-error
+from flask import request  # pylint: disable=import-error
+from flask import jsonify  # pylint: disable=import-error
+
+from parser import parse_all_documents  # pylint: disable=import-error
+from connector import get_articles_that_match_keywords  # pylint: disable=import-error
+from connector import get_article_by_number  # pylint: disable=import-error
```

Comment:
There shouldn't be import errors here ... these files exist in the same directory. Can you please try to fix the import errors rather than ignoring them?